streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedTable.java

```diff
@@ -50,6 +55,8 @@
      *                      if not specified the default serdes defined in the configs will be used
      * @param name          the name of the resulted table
      * @param <T>           the value type of the aggregated {@link KTable}
+     * @return a {@link KTable} with same key but different value type as this {@link KGroupedTable}
```

Comment:
In practice `T` may still be the same as `V`, for example sum / max / min. So how about "with same key type and aggregated value type {@code T} containing ...". Ditto below.